ghcide/src/Development/IDE/Main.hs

```diff
@@ -268,6 +275,10 @@ defaultArguments recorder logger = Arguments
                 -- the language server tests without the redirection.
                 putStr " " >> hFlush stdout
                 return newStdout
+        , argsMonitoring = OpenTelemetry.monitoring
+#ifdef MONITORING_EKG
+                        <> EKG.monitoring logger 8999
```

Comment:
What about confining the CPP to `Development.IDE.Monitoring.EKG`. If ekg support is disabled, it could just export a no-op `Monitoring`.

Reply:
Good idea, thanks